tests/integration/cli/test_worklock_cli_functionality.py

```diff
@@ -491,22 +492,26 @@ def test_refund(click_runner,
     # Spy on the corresponding CLI function we are testing
     mock_refund = mocker.spy(Bidder, 'refund_deposit')
 
+    bidder_address = surrogate_bidder.checksum_address
     command = ('refund',
-               '--bidder-address', surrogate_bidder.checksum_address,
+               '--bidder-address', bidder_address,
                '--provider', MOCK_PROVIDER_URI,
-               '--network', TEMPORARY_DOMAIN,
-               '--force')
+               '--network', TEMPORARY_DOMAIN)
 
-    result = click_runner.invoke(worklock, command, input=INSECURE_DEVELOPMENT_PASSWORD, catch_exceptions=False)
+    user_input = INSECURE_DEVELOPMENT_PASSWORD + '\n' + YES
```

Comment:
`YES_ENTER` instead of `YES`? Or does it work like that as well?

Reply:
It works since it's the last input